_i18n/en/resources/user-guides/join-monero-matrix.md

```diff
@@ -0,0 +1,34 @@
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+The Monero community primarily communicates via IRC channels. We utilize these channels using the Matrix network. An open source network for secure, decentralized communication.
+
+In order to access the Matrix protocol you will need to use a Matrix client. The client we recommend is Element. Element is a free, open-source program that can be accessed via web, desktop, and mobile.
+
+### Getting Element
+
+Go to [element.io](https://element.io/get-started#download) to download the latest version of the Element app on your computer. Alternatively you can use the Element [web app](https://app.element.io). There are Android apps available on [F-Droid](https://f-droid.org/packages/im.vector.app/) or [Google Play](https://play.google.com/store/apps/details?id=im.vector.app). As well as an app for [iPhone](https://apps.apple.com/app/vector/id1083446067). Once you have accessed the client of your choice create an account on our host server.
+
+### Setting the custom host server
+
+While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted by the Monero core team to support decentralization. If you do choose this option remember to set your custom server when logging in on a new device.
+
+When signing up choose to edit your host server and type in `matrix.monero.social` as shown.
+
+Web/Desktop view
+
+![Web-Desktop-View](/img/resources/user-guides/en/join-monero-matrix/desktop-web.png){:width="600px"}
+
+Android view
+
+![iPhone-View](/img/resources/user-guides/en/join-monero-matrix/android.png){:height="600px"}
+
+iPhone View
+
+![Android-View](/img/resources/user-guides/en/join-monero-matrix/iphone.png){:height="600px"}
+
+### Creating your account
+
+You must provide a username and password to setup an account. We do not recommend providing a phone number for privacy reasons. The email field is used solely for recovering your account if you lose your password. 
+
+Once you have entered in your credentials continue through the setup steps, and log into your account. You should be free from there to chat on the available Monero channels!
```

Comment:
```suggestion
Once you have entered in your credentials continue through the setup steps, and log into your account. Now search for "Monero" rooms and join the communities you like!
```

or something along these lines^. So that people know the next step after logging in is to look for Monero rooms.